src/driver/main.cpp

```diff
@@ -455,8 +455,24 @@ struct compiler
     {
         auto p = l.load();
         // Dont compile if its already been compiled
+
         if(p.is_compiled())
+        {
+
+            if(ct.target_name == "gpu" && is_offload_copy_set(p))
+            {
+                std::cout << "Compiled MIGraphX program likely has offload_copy set, Try passing "
+                             "`--enable-offload-copy` if program run fails.\n";
+            }
+            else if(ct.target_name == "gpu")
```

Comment:
```suggestion
            if(ct.target_name == "gpu" and is_offload_copy_set(p) and not co.offload_copy)
            {
                std::cout << "Compiled MIGraphX program likely has offload_copy set, Try passing "
                             "`--enable-offload-copy` if program run fails.\n";
            }
            else if(ct.target_name == "gpu" and co.offload_copy)
```
I think we should check what the offload copy flag is set to before printing any messages.